index.js

```diff
@@ -1,10 +1,12 @@
 /*! torrent-discovery. MIT License. WebTorrent LLC <https://webtorrent.io/opensource> */
-const debug = require('debug')('torrent-discovery')
-const DHT = require('bittorrent-dht/client') // empty object in browser
-const EventEmitter = require('events').EventEmitter
-const parallel = require('run-parallel')
-const Tracker = require('bittorrent-tracker/client')
-const LSD = require('bittorrent-lsd')
+import debugFactory from 'debug'
+import DHT from 'bittorrent-dht/client.js' // empty object in browser
+import { EventEmitter } from 'events'
+import parallel from 'run-parallel'
+import Tracker from 'bittorrent-tracker/client.js'
+import LSD from 'bittorrent-lsd'
+
+const debug = debugFactory('torrent-discovery')
```

Comment:
```suggestion
import Debug from 'debug'
import DHT from 'bittorrent-dht/client.js' // empty object in browser
import { EventEmitter } from 'events'
import parallel from 'run-parallel'
import Tracker from 'bittorrent-tracker/client.js'
import LSD from 'bittorrent-lsd'

const debug = debug('torrent-discovery')
```

Reply:
this is a perfect example of why I didn't want to do this, Debug is too close to debug and its very easy to mistake the 2 like you just did

Reply:
True, but capital makes it obvious, just like with other imports

Reply:
no other compromises here? @DiegoRBaquero